src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -120,6 +131,33 @@ def test_conversion_as_lone_interface(self):
                 self.run_custom_checks()
 
 
+def _create_mock_probe(*, num_channels: int):
```

Comment:
```suggestion
def _create_mock_probe(num_channels: int):
```

We don't tend to hard enforce keyword argument usage with input validation; if we started to, we should do it elsewhere but it would need to be discussed with the rest of the team first

Reply:
As I said in the other comment, I think it is dangerous to not enforce it, for future-proofing backward compatibility. But for the sake of this PR I have removed the `*`.